source/text/label.ts

```diff
@@ -16,190 +16,77 @@ import { Text } from './text';
     }
 
-    /**
-     * Returns, whether or not the character at a given index is equal to the default or the text's line feed character.
-     * @param index - The zero-based index of the desired character. If there is no character at the specified index,
-     * NaN is returned.
-     * @returns - true if char at given index equals the text's line feed character
-     */
-    lineFeedAt(index: number): boolean {
-        return this.charAt(index) === this.lineFeed;
+    set wrap(flag: boolean) {
+        this._wrap = flag;
     }
-
-
-    /**
-     * Gets the kerning value before (i.e., left in left-to-right writing systems) the given glyph index.
-     * @param index - index of the glyph in this label
-     * @returns - kerning value before glyph at given index
-     */
-    kerningBefore(index: number): number {
-        if (index < 1 || index > this.length) {
-            return NaN;
-        }
-        return this._fontFace!.kerning(this.charCodeAt(index - 1), this.charCodeAt(index));
+    get wrap(): boolean {
+        return this._wrap;
     }
 
-    /**
-     * Gets the kerning value after (i.e., right in left-to-right writing systems) the given glyph index.
-     * @param index - index of the glyph in this label
-     * @returns - kerning value after glyph at given index
-     */
-    kerningAfter(index: number): number {
-        if (index < 0 || index > this.length - 1) {
-            return NaN;
-        }
-        return this._fontFace!.kerning(this.charCodeAt(index), this.charCodeAt(index + 1));
+    set elide(elide: Label.Elide) {
+        this._elide = elide;
     }
-
-    /**
-     * Returns the advancement of the glyph at given index.
-     * @param index - The zero-based index of the desired character. If there is no character at the specified index,
-     * NaN is returned.
-     * @returns - advancement of the glyph at given index or NaN
-     */
-    advance(index: number): number {
-        if (index < 0 || index > this.length) {
-            return NaN;
-        }
-        return this._fontFace!.glyph(this.charCodeAt(index)).advance;
+    get elide(): Label.Elide {
+        return this._elide;
     }
 
-
-    /**
-     * Text that is to be rendered.
-     */
-    set text(text: Text | string) {
-        this._altered.alter('text');
-        this._text = text;
+    set ellipsis(ellipsis: string) {
+        this._ellipsis = ellipsis;
     }
-    get text(): Text | string {
-        return this._text;
+    get ellipsis(): string {
+        return this._ellipsis;
     }
 
-    /**
-     * Length of the text, i.e., number of characters within the text.
-     */
-    get length(): number {
-        return this._text.length;
-    }
 
-    /**
-     * Character that is to be used for Line feed.
-     */
-    get lineFeed(): string {
-        if (this._text instanceof Text) {
-            return this._text.lineFeed;
-        }
-        return Text.DEFAULT_LINEFEED;
+    set lineWidth(lineWidth: number) {
```

Comment:
missig doc.
it is important that the setter sets the line width in Label.SpaceUnit, while the getter returns the line width in Typesetting unit.